lighty-core/dependency-versions/pom.xml

```diff
@@ -129,7 +129,7 @@
             <dependency>
                 <groupId>org.junit</groupId>
                 <artifactId>junit-bom</artifactId>
```

Comment:
Let's remove this commit for now. It will be part of another PR.

Reply:
commit droped